src/astra_driver.cpp

```diff
@@ -717,6 +720,7 @@ sensor_msgs::msg::CameraInfo::SharedPtr AstraDriver::getDepthCameraInfo(int widt
 
 void AstraDriver::readConfigFromParameterServer()
 {
+  depth_frame_id_ = std::string("openni_depth_optical_frame");
```

Comment:
I got that particular name out of the ROS1 astra camera driver, where they attempt to get the name for the depth_frame_id_ out of the parameter server and fall back to to "/openni_depth_optical_frame" if it doesn't exist.  I don't have any particular affinity for that name, so I can change it if you feel like we should.